pkg/scheduler/framework/runtime/framework.go

```diff
@@ -676,15 +676,46 @@ func (f *frameworkImpl) QueueSortFunc() framework.LessFunc {
 // When it returns Skip status, returned PreFilterResult and other fields in status are just ignored,
 // and coupled Filter plugin/PreFilterExtensions() will be skipped in this scheduling cycle.
 // If a non-success status is returned, then the scheduling cycle is aborted.
-func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (_ *framework.PreFilterResult, status *framework.Status) {
+func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod, diagnosis *framework.Diagnosis, allNodes []*framework.NodeInfo) (_ *framework.PreFilterResult, status *framework.Status) {
 	startTime := time.Now()
 	skipPlugins := sets.New[string]()
+	message := func(plugins []string) string {
+		if len(plugins) == 0 {
+			return ""
+		}
+		msg := fmt.Sprintf("node(s) didn't satisfy plugin(s) %v simultaneously", plugins)
+		if len(plugins) == 1 {
+			msg = fmt.Sprintf("node(s) didn't satisfy plugin %v", plugins[0])
+		}
+		return msg
+	}
+	preFilterNodeToStatusMap := framework.NodeToStatusMap{}
```

Comment:
Instead of using `preFilterNodeToStatusMap`, can we directly pass `diagnosis` to `registerPreFilterPlugins()`?

Reply:
already use `diagnosis.NodeToStatusMap` instead of `preFilterNodeToStatusMap `